### ISSUE TITLE
**Bus Fault**

### ISSUE THREAD

**Issue by cmoorice** [fhorinek/BB#142]
updated to 208, everything seems fine. However, cannot turn on anymore and have the **Bus Fault** page after charging, please help.

[config.zip](https://github.com/fhorinek/BB/files/7787092/config.zip)
.

**Comment by fhorinek** [comment]
What exactly do you see?
Can you send me the picture?

You can try full wipe.
https://youtu.be/8RftdomWoWA
This will erase everything from the device. You will need install firmware manually. Place strato.fw from strato.skybean.eu/fw/208 to the SD card

**Comment by cmoorice** [comment]
Dear Frantisek,

Thank you so much for your prompt reply. I did the full wipe before I
received your email. However, after the full wipe and re-installation of
the fw, everything seems fine now. I tried to use the strato yesterday and
up to now, no "Bus Fault" screen appears again. I guess the initial update
may be corrupted, or something else that caused the previous issue.

Thanks again for your help and Happy New Year!

Best regards,
Maurice
